src/content/docs/alerts-applied-intelligence/notifications/intro-notifications.mdx

```diff
@@ -1,25 +1,43 @@
 ---
-title: Introduction to notifications
+title: Introduction to alert notifications
 tags:
   - Alerts and applied intelligence 
   - Applied intelligence
   - Notifications
   - Workflows
   - Error Inbox
-metaDescription: "Notifications are where you integrate your New Relic data with third-party services, such as Slack, Email, Jira, and ServiceNow."
+metaDescription: "Notifications are how you integrate your New Relic data with third-party services, such as Slack, Email, Jira, and ServiceNow."
 ---
 
-Notifications are a consolidation of the different ways to send notification-events to third-party services, such as Slack, Jira, ServiceNow, and email. You can also use webhooks to send your data to any compatible third-party service.
+DevOps teams have a lot to monitor as systems grow increasingly complex and it can be hard to keep track of what is going on, where. Notifications is a suite of tools that enable you to get the right information to the right people as soon as possible. Notifications send notification-events to third-party services, such as Slack, Jira, ServiceNow, and email. You can also use webhooks to send your data to any compatible third-party service.
```

Comment:
Should "notification-events" be hyphenated? My instinct says no, but I can't explain why.

Reply:
Done!